src/main/java/me/jellysquid/mods/sodium/client/world/WorldSlice.java

```diff
@@ -227,11 +230,15 @@ public boolean isAirBlock(int x, int y, int z) {
 
     @Override
     public BiomeGenBase getBiomeGenForCoords(int x, int z) {
+        if (!blockBoxContains(this.volume, x, 0, z)) {
```

Comment:
You should probably use `this.volume.minX` or some valid Y value here, otherwise I think most biome retrievals will fail.